current_year.md

```diff
@@ -4,6 +4,4 @@ title: Interacting Minds
 year: 2020
 ---
 
-This year's 2020 MIND summer school will be on Interacting MINDS from **8/5/19-8/15-19**. 
-
-Cognitive maps are most commonly associated with the representation and navigation of physical space, but the concept of a map is more generally concerned with how experience is organized to form knowledge. Tantalizingly, it seems that the neural encoding of physical space may also underlie the organization of conceptual and social spaces. The 2019 MIND Computational Summer School will provide participants with tools to study what principles govern the learning, organization, and application of such maps, and to probe their neural underpinnings.
+The bulk of research in psychology and neuroscience has focused on studying processes within single individuals. However, outside of the laboratory people rarely operate independently but rather in the context of a complex constellation of social relationships. Social interactions form a critical aspect of the human experience from romantic relationships, families, to professional relationships, and everyday transactions such as commuting to work and buying groceries, yet have received surprisingly little empirical study. This year's 2020 MIND Computational Summer School will take place between **8/5/19-8/15-19** and will focus on understanding how ***minds interact*** and connect with each other. We will introduce computational techniques to help us understand how we infer another person's internal beliefs, goals, intentions, and feelings and also how we communicate this information to others. We will cover topics ranging from communication, mentalizing, synchronization, social networks, and more.
```

Comment:
Minor suggestions and a couple of additions to make it clear we are interested in making connections with animal work:

```
The bulk of research in psychology and neuroscience has focused on studying processes within single individuals. However, outside of the laboratory, individuals rarely operate independently but rather in the context of a complex constellation of social relationships. Social interactions form a pervasive and critical aspect of daily life, in settings such as professional and romantic relationships, family interactions, and everyday transactions such as commuting to work and buying groceries. Yet, these settings have received surprisingly little empirical study. This year's 2020 MIND Computational Summer School will take place between **8/5/19-8/15-19** and will focus on understanding how ***minds interact*** and connect with each other. We will introduce computational techniques to help us understand how we infer the beliefs, goals, intentions, and feelings of other agents, and how this information is communicated to others. Increasingly, it is becoming possible to study these processes in non-human animals, resulting in a productive synergy between researchers working with human and non-human subjects. We will cover topics such as communication, mentalizing, synchronization, social networks, and more.
```